docker/Dockerfile

```
@@ -0,0 +1,24 @@
+# Created by Tomas Kolarik
+# Eternal glory to him
+# Eternal suffering for every mistake
+
+FROM debian:latest
+USER root
+
+RUN apt-get update -y
+RUN apt-get install -y \
+    apt-utils \
+    make \
+    cmake \
+    build-essential \
+    git \
+    bison \
+    flex \
+    libgmp-dev
+
+RUN git clone https://github.com/usi-verification-and-security/opensmt.git --single-branch
```

```suggestion
RUN git clone --depth 1 https://github.com/usi-verification-and-security/opensmt.git
```

We don't need the whole history, no? Just the latest HEAD.

---

Sure!